### ISSUE TITLE
Threema::Receive::Image is never reached

### ISSUE THREAD

**Issue by roschaefer** [thorsteneckel/threema#18]
This is a follow-up of #17

During debugging I noticed that no matter if I send an image as "file" or as "image", this class is never instantiated:
https://github.com/thorsteneckel/threema/blob/master/lib/threema/receive/image.rb

Instead it's always `Threema::Receive::File` that's being created.

Here's what I do:
![Swappshot Mon Feb 15 13:03:32 2021](https://user-images.githubusercontent.com/2110676/107944345-3fb81580-6f8e-11eb-97f8-a34da830ad7b.png)

@thorsteneckel can you confirm that the class did serve it's purpose in the past?

**Comment by thorsteneckel** [comment]
Yes, I can confirm that I've sent and received cat images in the past to test this. However, it has been a while and Threema was super productive in the meantime while this repository didn't receive any updates.

**Comment by rugk** [comment]
AFAIK the Threema app itself also only uses file messages and has deprecated image messages. Also the Threema Gateway API does not support images or video messages anymore.

At 2020-10-29 there was a message about that:
> Liebe Kundin, lieber Kunde
>
>In Zukunft werden die Threema-Apps Bilder und Videos als Datei-Nachrichten und nicht mehr wie bisher als Bild- bzw. Video-Nachrichten versenden.
>
>Für App-Nutzer bleibt alles beim Alten. Sollte aber Ihre Gateway-Anwendung voraussetzen, dass Nachrichten des Typs «Bild» eingehen, ist eine Erweiterung erforderlich, welche Nachrichten des Typs «Datei» zulässt (vgl. https://gateway.threema.ch/de/developer/api#E2EMessageFormat).
>
>Als Erstes führt Threema für iOS diese Neuerung ein. Das entsprechende Update auf Version 4.6.3 wird voraussichtlich nächste >oder übernächste Woche erscheinen. Threema für Android und die Threema Work-Apps werden später folgen.
>
>Bei Fragen stehen wir gerne zu Ihrer Verfügung.
>
>Freundliche Grüsse
>Threema
